backend/src/packages/shifts/shift.service.ts

```diff
@@ -66,6 +66,35 @@ class ShiftService implements IService {
     return shifts.map((it) => ShiftEntity.initialize(it).toObject());
   }
 
+  public async findOpenedByDriver(
+    driverId: number,
+  ): Promise<ShiftResponseDto | null> {
+    const shift = await this.shiftRepository.getOpenedByDriver(driverId);
+
+    return shift ? ShiftEntity.initialize(shift).toObject() : null;
+  }
+
+  public async findOpenedByTruckWithBusiness(
+    truckId: number,
+  ): Promise<
+    (ShiftEntityT & { businessId: number; driverLicenseNumber: string }) | null
+  > {
+    const shiftDatabase =
+      await this.shiftRepository.getOpenedByTruckWithBusiness(truckId);
+
+    if (shiftDatabase) {
```

Comment:
It'll look simpler if we immediately return null: if(!shiftDatabase){return null;}